README.md

```diff
@@ -64,13 +70,20 @@ npm t
 #### End-to-end tests
 To run e2e tests for the sandbox, you need to supply an environment. A `local` environment and an environment template are included under `tests/e2e/environments`.
 
-In order for local tests to work, you must have the sandbox server running locally.
+In order for tests under the make target `test-sandbox` to work, you must have the sandbox server running locally. In a seperate shell run:
 
 ```
 make run-sandbox
```

Comment:
As the "run" command has a dependency on the "build" command, is it worth mentioning here to run it first? Likewise for the coordinator